staging/src/k8s.io/kubectl/pkg/cmd/set/helper.go

```diff
@@ -69,7 +70,7 @@ func selectString(s, spec string) bool {
 		// last part does not exactly match remaining part of string
 		case i == (len(parts)-1) && len(s) != (len(part)+next):
 			match = false
-			break
+			break Loop
```

Comment:
Why is this needed, this works just fine and is only needed when two or more loops is in effect.

Reply:
There is no need to add a break in switch cases for go. I think the logic here is to skip the for loop once the `match` variable is set as false.